README.md

```diff
@@ -48,6 +91,14 @@ As with any GitHub workflow, you have all the power — all the action needs is
 
 Once you’ve configured and committed your workflow to the default branch of your repository, you’re all set, and can now [return to your Stark project](https://account.getstark.co/projects) and run accessibility scans.
 
-## Implementation Details
+## More Examples
+
+We have several repositories available that demonstrate how you can configure your workflow for popular web tooling:
+
+- [Vite](https://github.com/stark-projects-demos/vite-demo)
+- [Next.js](https://github.com/stark-projects-demos/nextjs-demo)
+- [Gatsby](https://github.com/stark-projects-demos/gatsby-demo)
+
+## Technical Details
```

Comment:
Anything we need to add for permissions here (just double-checking): 
https://github.com/actions/labeler#permissions

Reply:
Pretty sure the answer is no, but cc: @abcdefghiraj. (If we do need a section like that, I think it can be added in a follow-up pull request.)